Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Requirement already satisfied: numpy>=1.17.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 1)) (1.24.1)
Requirement already satisfied: torch>=1.4.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (1.13.1)
Requirement already satisfied: torchvision>=0.4.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 3)) (0.14.1)
Requirement already satisfied: visdom>=0.1.6 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (0.2.3)
Requirement already satisfied: Pillow==6.2.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (6.2.0)
Requirement already satisfied: tensorboard==2.11.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (2.11.0)
Requirement already satisfied: python-mnist in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from -r requirements.txt (line 7)) (0.7)
Requirement already satisfied: google-auth<3,>=1.6.3 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (2.16.0)
Requirement already satisfied: setuptools>=41.0.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (65.6.3)
Requirement already satisfied: google-auth-oauthlib<0.5,>=0.4.1 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (0.4.6)
Requirement already satisfied: markdown>=2.6.8 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (3.4.1)
Requirement already satisfied: protobuf<4,>=3.9.2 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (3.20.3)
Requirement already satisfied: tensorboard-data-server<0.7.0,>=0.6.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (0.6.1)
Requirement already satisfied: tensorboard-plugin-wit>=1.6.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (1.8.1)
Requirement already satisfied: grpcio>=1.24.3 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (1.51.1)
Requirement already satisfied: wheel>=0.26 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (0.37.1)
Requirement already satisfied: werkzeug>=1.0.1 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (2.2.2)
Requirement already satisfied: absl-py>=0.4 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (1.4.0)
Requirement already satisfied: requests<3,>=2.21.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from tensorboard==2.11.0->-r requirements.txt (line 6)) (2.28.2)
Requirement already satisfied: nvidia-cuda-nvrtc-cu11==11.7.99 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from torch>=1.4.0->-r requirements.txt (line 2)) (11.7.99)
Requirement already satisfied: nvidia-cublas-cu11==11.10.3.66 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from torch>=1.4.0->-r requirements.txt (line 2)) (11.10.3.66)
Requirement already satisfied: nvidia-cuda-runtime-cu11==11.7.99 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from torch>=1.4.0->-r requirements.txt (line 2)) (11.7.99)
Requirement already satisfied: typing-extensions in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from torch>=1.4.0->-r requirements.txt (line 2)) (4.4.0)
Requirement already satisfied: nvidia-cudnn-cu11==8.5.0.96 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from torch>=1.4.0->-r requirements.txt (line 2)) (8.5.0.96)
Requirement already satisfied: jsonpatch in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from visdom>=0.1.6->-r requirements.txt (line 4)) (1.32)
Requirement already satisfied: six in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from visdom>=0.1.6->-r requirements.txt (line 4)) (1.16.0)
Requirement already satisfied: websocket-client in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from visdom>=0.1.6->-r requirements.txt (line 4)) (1.4.2)
Requirement already satisfied: networkx in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from visdom>=0.1.6->-r requirements.txt (line 4)) (3.0)
Requirement already satisfied: scipy in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from visdom>=0.1.6->-r requirements.txt (line 4)) (1.10.0)
Requirement already satisfied: tornado in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from visdom>=0.1.6->-r requirements.txt (line 4)) (6.2)
Requirement already satisfied: rsa<5,>=3.1.4 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from google-auth<3,>=1.6.3->tensorboard==2.11.0->-r requirements.txt (line 6)) (4.9)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from google-auth<3,>=1.6.3->tensorboard==2.11.0->-r requirements.txt (line 6)) (5.3.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from google-auth<3,>=1.6.3->tensorboard==2.11.0->-r requirements.txt (line 6)) (0.2.8)
Requirement already satisfied: requests-oauthlib>=0.7.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard==2.11.0->-r requirements.txt (line 6)) (1.3.1)
Requirement already satisfied: importlib-metadata>=4.4 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from markdown>=2.6.8->tensorboard==2.11.0->-r requirements.txt (line 6)) (6.0.0)
Requirement already satisfied: idna<4,>=2.5 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from requests<3,>=2.21.0->tensorboard==2.11.0->-r requirements.txt (line 6)) (3.4)
Requirement already satisfied: charset-normalizer<4,>=2 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from requests<3,>=2.21.0->tensorboard==2.11.0->-r requirements.txt (line 6)) (3.0.1)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from requests<3,>=2.21.0->tensorboard==2.11.0->-r requirements.txt (line 6)) (1.26.14)
Requirement already satisfied: certifi>=2017.4.17 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from requests<3,>=2.21.0->tensorboard==2.11.0->-r requirements.txt (line 6)) (2022.12.7)
Requirement already satisfied: MarkupSafe>=2.1.1 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from werkzeug>=1.0.1->tensorboard==2.11.0->-r requirements.txt (line 6)) (2.1.2)
Requirement already satisfied: jsonpointer>=1.9 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from jsonpatch->visdom>=0.1.6->-r requirements.txt (line 4)) (2.3)
Requirement already satisfied: zipp>=0.5 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from importlib-metadata>=4.4->markdown>=2.6.8->tensorboard==2.11.0->-r requirements.txt (line 6)) (3.11.0)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard==2.11.0->-r requirements.txt (line 6)) (0.4.8)
Requirement already satisfied: oauthlib>=3.0.0 in /root/anaconda3/envs/py38_new/lib/python3.8/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard==2.11.0->-r requirements.txt (line 6)) (3.2.2)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
